tests/install-deps.sh

```diff
@@ -2,6 +2,16 @@
 
 # Helper script for Continuous Integration Services
 
+# Check if the last commit message requests the CI to skip the build.
+git log HEAD~..HEAD > last-commit
```

Comment:
I might not understand the git magic but I'll warn anyway: 
this might not be what we want. Imagine a PR that has code changes and doc changes in separate commits. The doc changing commit is HEAD and contains the `[ skip ci ]` keyword, now CI is skipped altogether.